<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mlMainWindow</class>
 <widget class="QMainWindow" name="mlMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mollana</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mollana.qrc">
    <normaloff>:/icons/png32/mollana64.png</normaloff>:/icons/png32/mollana64.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="mlMainWindowCentral">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="tbxEditor">
      <property name="font">
       <font>
        <family>Droid Sans Mono</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="actionGotoLine"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <widget class="QMenu" name="menuFormatFont">
     <property name="title">
      <string>Font</string>
     </property>
     <addaction name="actionEditorFont"/>
     <addaction name="actionUnicodeViewFont"/>
    </widget>
    <addaction name="actionWordWrap"/>
    <addaction name="menuFormatFont"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionUnicodeView"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGettingStarted"/>
    <addaction name="actionCheatSheet"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Standard</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
  </widget>
  <widget class="QDockWidget" name="dckUnicodeView">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Unicode View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dckContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextEdit" name="tbxUnicodeView">
       <property name="font">
        <font>
         <family>Droid Arabic Naskh</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/new.png</normaloff>:/icons/png32/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>Create new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/add_to_open_folder.png</normaloff>:/icons/png32/add_to_open_folder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open a text file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/save.png</normaloff>:/icons/png32/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/transfer.png</normaloff>:/icons/png32/transfer.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save as a different file</string>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/shut_down.png</normaloff>:/icons/png32/shut_down.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/blue_arrow_down.png</normaloff>:/icons/png32/blue_arrow_down.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/blue_arrow_up.png</normaloff>:/icons/png32/blue_arrow_up.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/copy_paste.png</normaloff>:/icons/png32/copy_paste.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/cut.png</normaloff>:/icons/png32/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/phone_book_edit.png</normaloff>:/icons/png32/phone_book_edit.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/accept_page.png</normaloff>:/icons/png32/accept_page.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGotoLine">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/search_page.png</normaloff>:/icons/png32/search_page.png</iconset>
   </property>
   <property name="text">
    <string>Goto Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionWordWrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/refresh_page.png</normaloff>:/icons/png32/refresh_page.png</iconset>
   </property>
   <property name="text">
    <string>Word Wrap</string>
   </property>
   <property name="toolTip">
    <string>Toggle word wrap</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionUnicodeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/office_folders.png</normaloff>:/icons/png32/office_folders.png</iconset>
   </property>
   <property name="text">
    <string>Unicode View</string>
   </property>
   <property name="toolTip">
    <string>Toggle unicode output window</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/help_balloon.png</normaloff>:/icons/png32/help_balloon.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionEditorFont">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/text_page.png</normaloff>:/icons/png32/text_page.png</iconset>
   </property>
   <property name="text">
    <string>Editor Font</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionUnicodeViewFont">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/chalk_board.png</normaloff>:/icons/png32/chalk_board.png</iconset>
   </property>
   <property name="text">
    <string>Unicode View</string>
   </property>
   <property name="toolTip">
    <string>Change Unicode View font</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionGettingStarted">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/help.png</normaloff>:/icons/png32/help.png</iconset>
   </property>
   <property name="text">
    <string>Getting Started</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="actionCheatSheet">
   <property name="icon">
    <iconset resource="mollana.qrc">
     <normaloff>:/icons/png32/office_folders.png</normaloff>:/icons/png32/office_folders.png</iconset>
   </property>
   <property name="text">
    <string>Cheat Sheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mollana.qrc"/>
 </resources>
 <connections/>
</ui>
